owners/test/owners.test.js

```diff
@@ -128,8 +128,9 @@ describe('owners tree', () => {
 
     it('returns the nearest tree with a rule', () => {
       tree.addRule(childDirRule);
+      tree.addRule(descendantDirRule);
 
-      expect(tree.atPath('foo/bar/baz').dirPath).toEqual('foo');
+      expect(tree.atPath('foo/bar/okay.txt').dirPath).toEqual('foo');
```

Comment:
Can you explain this change?

Reply:
The test was meant to check that it would return the nearest tree with a rule, as opposed to returning some deep node that has no rules. But `atPath` short-circuits once the deepest owners tree is reached, so it wasn't actually testing the right bit. This makes sure that there is a subtree that exists for `foo/bar`, but it is skipped because it has no rules.